cf_xarray/tests/test_accessor.py

```diff
@@ -1233,3 +1233,34 @@ def test_cmip6_attrs():
     )
     assert da.cf.axes["X"] == ["nlon"]
     assert da.cf.axes["Y"] == ["nlat"]
+
+
+def test_custom_criteria():
+    my_custom_criteria = {
+        "ssh": {
+            "standard_name": (
+                "sea_surface_height_amplitude_due_to_geocentric_ocean_tide_geoid_mllw",
+                "sea_surface_height_above_sea_level_geoid_mllw",
+            ),
+            "name": ("sea_surface_elevation",),  # variable name
+        },
+        "temp": {
+            "standard_name": ("sea_water_temperature",),
+        },
+        "salinity": {
+            "standard_name": ("sea_water_salinity",),
+        },
+        "u": {
+            "standard_name": ("eastward_sea_water_velocity",),
+        },
+        "v": {
+            "standard_name": ("northward_sea_water_velocity",),
+        },
+        "wind_speed": {
+            "standard_name": ("wind_speed",),
+        },
+        "wind_speed_of_gust": {
+            "standard_name": ("wind_speed_of_gust",),
+        },
+    }
+    cf_xarray.accessor.set_options(my_custom_criteria)
```

Comment:
I think if you add your tests here, we should be done.

Reply:
I will work on tests now.